Blog: Update Product Engineer vs Software Engineer (#7804)
PostHog/posthog.com: contents/blog/product-engineer-vs-software-engineer.md

tags:
  - Explainers
  - Product engineers
crosspost:
  - Product engineers
---

Software companies were once dominated by two types of roles: product managers and software engineers, but this is changing. A new role is emerging: the [product engineer](/blog/what-is-a-product-engineer).

Want proof? Look at this graph:

![product-engineer-trends](../images/blog/product-engineer-trend.png)

> Online searches for product engineers saw a big and sustained jump in 2022.

Reviewer: ```suggestion
Online searches for product engineers have grown by 80% since 2021 as companies, particularly startups like ourselves, [incident.io](http://incident.io/),  and [Ghost](https://ghost.org/), prioritize engineers who are customer obsessed, talk to them directly, and comfortable working autonomously. 
```

Product engineers still spend the majority of their time coding, but they have responsibilities outside of coding that software engineers don't.

> Product engineers also spend their time talking to customers, digging into usage data, and researching the competitive landscape. They test other products, build prototypes, and brainstorm experiments to make their product better. They use this research to prioritize a roadmap that they own. Along with this is responsibility to explain the roadmap and get feedback from users and the company.

Reviewer: ```suggestion
Product engineers also spend their time talking to customers, digging into usage data, and researching the competitive landscape. They test other products, build prototypes, and brainstorm experiments to make their product better. They use this research to prioritize a roadmap that they own, and take responsibility for explaining the roadmap and gathering feedback from users and the company.
```
